backend/src/main/java/hu/flowacademy/meteo/controller/StationController.java

```diff
@@ -0,0 +1,24 @@
+package hu.flowacademy.meteo.controller;
+
+import hu.flowacademy.meteo.dto.StationDto;
+import hu.flowacademy.meteo.service.StationService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequiredArgsConstructor
+public class StationController {
+
+    private final StationService stationService;
+
+    @CrossOrigin("*")
```

Comment:
Nem kell CrossOrigin a proxy miatt. *-ot meg amúgy sem állítunk be, az nagyon insecure.